src/parser/jobs/nin/Utility.js

```diff
@@ -93,11 +96,11 @@ export default class Utility extends Module {
 			this.suggestions.add(new Suggestion({
 				icon: ACTIONS.SHADEWALKER.icon,
 				content: <Fragment>
-					<ActionLink {...ACTIONS.SHADEWALKER}/> should always be used on the pull, as it will help your main tank solidify threat much more quickly and reduce everyone else&apos;s risk of stealing threat.
+					<Trans id="nin.utility.suggestions.no-shade.content"><ActionLink {...ACTIONS.SHADEWALKER}/> should almost always be used on the pull, as it will help your main tank solidify threat much more quickly and reduce everyone else&apos;s risk of stealing threat.</Trans>
```

Comment:
P.S. I added "almost" in here specifically because of Rathalos, the little shit.

Reply:
R I P